src/WebJobs.Script.WebHost/Management/MeshInitServiceClient.cs

```diff
@@ -42,21 +44,58 @@ public async Task MountCifs(string connectionString, string contentShare, string
         public Task MountBlob(string connectionString, string contentShare, string targetPath)
         {
             // todo: Implement once mesh init server supports mounting blobs
-            throw new NotImplementedException();
+            throw new NotImplementedException(nameof(MountBlob));
         }
 
         public async Task MountFuse(string type, string filePath, string scriptPath)
-            => await Mount(new[]
+            => await SendAsync(new[]
             {
                 new KeyValuePair<string, string>("operation", type),
                 new KeyValuePair<string, string>("filePath", filePath),
                 new KeyValuePair<string, string>("targetPath", scriptPath),
             });
 
-        private async Task Mount(IEnumerable<KeyValuePair<string, string>> formData)
+        public async Task PublishContainerFunctionExecutionActivities(IEnumerable<ContainerFunctionExecutionActivity> activities)
         {
-            var res = await _client.PostAsync(_environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI), new FormUrlEncodedContent(formData));
+            var meshInitServiceUri = _environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI);
```

Comment:
consider moving this to a general request start log in SendAsync, that way we're logging for all requests.